src/theory/bv/theory_bv_rewrite_rules_operator_elimination.h

```diff
@@ -19,20 +19,22 @@
 
 #pragma once
 
+#include "options/bv_options.h"
```

Comment:
Why do you need this include here?

Reply:
Now that I am including `theory_bv_rewrite_rules_operator_elimination.h` in the test file `test/unit/theory/theory_bv_rewriter_white.h`, i needed to added this include so `make units` works. Should I put the inclusion elsewhere?

Reply:
If options are need in `theory_bv_rewrite_rules_operator_elimination.h` then it should be included there.

Reply:
Then this particular issue is resolved?

Reply:
Ah sure... This is the header file, I was confused. Then it's resolved.